openml/config.py

```diff
@@ -252,11 +252,11 @@ def _setup(config: _Config | None = None) -> None:
     if config is None:
         config = _parse_config(config_file)
 
-    avoid_duplicate_runs = config.get("avoid_duplicate_runs", False)
+    avoid_duplicate_runs = bool(config.get("avoid_duplicate_runs", False))
```

Comment:
This is not going to work, `bool("False")` will resolve to `True`.

Reply:
changed to ```avoid_duplicate_runs = config.get("avoid_duplicate_runs", "").lower() == "true"```